stable/anchore-engine/values.yaml

```diff
@@ -556,6 +556,16 @@ anchoreCatalog:
     # Note: if a runtime inventory image's digest is also in anchore for regular image analysis, it won't be removed.
     image_ttl_days: 1
 
+    # Since Anchore is running in Kubernetes, we can collect runtime inventory data out of the box
+    kubernetes:
```

Comment:
Is this nesting necessary? Generally for chart configs you want to keep things as flat as possible. Also generally want to use camel casing in values.yaml unless keys are directly exposed to the app config.

Can it be something like:
```
deploymentInventory:
  enabled: true
  reportedClusterName: anchore-k8s
  namespaces:
    - all

Reply:
fixed!